example/per.jonprl

```diff
@@ -0,0 +1,125 @@
+Operator per-squash : (0).
+Prefix 7 "⇃" := per-squash.
+[⇃ A] =def= [per(lam(_.lam(_.A)))].
```

Comment:
^^ (To be clear, they're not equivalent in the sense of being the same PER, but are in the sense of having the same truth conditions, I think.)